tools/update-optional-dependencies.py

```diff
@@ -64,7 +64,7 @@ def from_tuple(cls, *decls: t.Any) -> Dependencies:
 
 
 _BENTOML_EXT = ["grpc", "io"]
-_TRANSFORMERS_EXT = ["torch", "tokenizers", "accelerate"]
+_TRANSFORMERS_EXT = ["torch", "tokenizers", "accelerate", "ctransformers"]
```

Comment:
There is no `ctransformers` for `transformers`. This `transformers_ext` is `transformers[torch,toeknizers,accelerate]` from pip